### PR TITLE
New what's new content

### PR DESCRIPTION
**Pull request by coreycaitlin**
Fixes issue(s) #1967 🎉 

[CircleCI](https://circleci.com/gh/18F/doi-extractives-data/tree/whats-new)

[🐥  PREVIEW 🐥 ](https://federalist.18f.gov/preview/18F/doi-extractives-data/whats-new/)

Changes proposed in this pull request:

- New content!
- The old content is currently commented out at the end of the page for reference
- This page might be too long and needs revision
- I might have forgotten some stuff

/cc @meiqimichelle

### CODE REVIEW

File: pages/whats_new.md [hunk 1]
```diff
@@ -7,22 +7,54 @@ breadcrumb:
...
-* **Reconciliation data:** A number of companies were asked to submit data for reconciliation with Department of Interior revenue data. Of the 45 companies invited, 35 participated. [Download reconciliation data.]({{ site.baseurl }}/downloads/reconciliation/)
-* **How it works:** Beyond the numbers, we've included new narratives explaining how the whole system of leasing federal land and collecting royalties on extracted materials works in the U.S. [Learn how it works.]({{ site.baseurl }}/how-it-works/)
+This version of the site includes new content, new state-level datasets, and completely new ways of exploring all data.
 
+## State and national profile pages
+
+In our research, users looked for information about their state and expressed interest in seeing data about their area across multiple datasets. New regional profile pages provide a more complete picture of extractive industries at the national and subnational levels. To see how this works, explore:
+
+* The [national profile page]({{ site.baseurl }}/explore/)
+* State profiles like [Texas]({{ site.baseurl }}/explore/), [Utah]({{ site.baseurl }}/explore/), or [California]({{ site.baseurl }}/explore/)
```
Comment (gemfarmer) [inline pages/whats_new.md]: these links don't go to the states, but to the national page!

File: pages/whats_new.md [hunk 2]
```diff
@@ -7,22 +7,54 @@ breadcrumb:
...
+* Offshore areas, like the [Gulf of Mexico]({{ site.baseurl }}/explore/offshore-gulf/) or [offshore Alaska areas]({{ site.baseurl }}/explore/offshore-alaska/)
+
+The profile pages also include more granular **disbursements data**, which shows how the Office of Natural Resources Revenue (ONRR) distributes revenue from the extractive industries to [national funds and tribes]({{ site.baseurl }}/explore/#federal-disbursements) and to state governments (for example, [Arizona]({{ site.baseurl }}/explore/AZ/#disbursements) or [Louisiana]({{ site.baseurl }}/explore/LA/#disbursements)).
+
+USEITI also invited states to opt in to the reporting process, with the goal of making state-level data more navigable for the public. In 2016, three states worked with the {{ "Independent Administrator" | term }} to provide **additional state-level data** and contextual information to be integrated in their state profile pages: [Alaska]({{ site.baseurl }}/explore/AK/), [Montana]({{ site.baseurl }}/explore/MT/), and [Wyoming]({{ site.baseurl }}/explore/WY/).
```
Comment (gemfarmer) [inline pages/whats_new.md]: I think Independent Administrator needs to be `{{ "Independent Administrator" | term: "Independent Administrator (IA)" }}`

File: pages/whats_new.md [hunk 3]
```diff
@@ -7,22 +7,54 @@ breadcrumb:
...
+
+## Contextual information
+
+The [How It Works]({{ site.baseurl }}/how-it-works/) section now has a question-and-answer format, along with several new elements:
+
+* **Reconciliation data:** This data was previously available, but we've redesigned the presentation to make it easier to understand and view the full results of the [USEITI reconciliation process]({{ site.baseurl }}/how-it-works/reconciliation/).
+* **New contextual information:** The {{ "Multi-stakeholder Group" | term }} approved new content about [the Coal Excise Tax]({{ site.baseurl }}/how-it-works/coal-excise-tax/), [Abandoned Mine Land Reclamation Program]({{ site.baseurl }}/how-it-works/aml-reclamation-program/), and [audits and assurances to ensure accurate revenue data]({{ site.baseurl }}/how-it-works/audits-and-assurances/).
```
Comment (gemfarmer) [inline pages/whats_new.md]: I think this similarly needs to be` {{ "Multi-stakeholder Group" | term: "Multi-stakeholder Group (MSG)" }}`

**Comment by gemfarmer** [comment]
Hey @coreycaitlin looks great. I noticed a few small things.

**Comment by meiqimichelle** [comment]
I like the groupings, and I think you pulled out the big changes perfectly! The only thing that stood out is the last section title: `Updates and design changes`. I wonder if there's something else to call that last bit that sounds less inclusive of everything on that page? Hmmmm....there's always the 'Other updates and design changes' approach :-/

I don't think it's too long!

**Comment by coreycaitlin** [comment]
Okay, I think I fixed those issues @gemfarmer — thanks for spotting them! And I added `Other` in that header, for now — maybe something more brilliant will occur to me later...

**Comment by gemfarmer** [comment]
LGTM!

**Comment by gemfarmer** [comment]
@meiqimichelle ?

**Pull request merged**